CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ if(Git_FOUND)
     endif()
 endif() #git
 
+include (ProcessorCount)
```

Comment:
This `include` should be moved to the file that calls `ProcessorCount`.

Reply:
there are actually two such files - `RippledSettings.cmake` and `RippledSanity.cmake`

Reply:
They should both include it. CMake modules have include guards when necessary just like C++ headers, and the best practice is the same: include what you use.